docs/source/examples/THCHS30.rst

```diff
@@ -1,328 +1,185 @@
-you need to load the :ref:`contribution:Catalog`.(L40)
-The catalog file ``catalog.json`` is in the same directory with dataloader file.
+.. code:: python
 
-In this example, there is no ``catalog.json`` file, because the lexion of ``THCHS-30`` is too
-large. The lexion is a list of dictionary lists. Therefore, We need to load subcatalog from the raw
-file and map it to have the catalog.(L57-63)
+    from tensorbay.dataset import Segment
 
-Add Data and Labels
--------------------
+    dev_segment = Segment("dev", dataset_client)
 
-It takes four key substeps to add the data and the labels:
+In the dev :ref:`reference/dataset_structure:Segment`,
+there is a sequence of :ref:`reference/dataset_structure:Data`.
+You can get one by index.
 
--  Creating :ref:`basic_concepts:Data` and adding content.
--  Creating :ref:`supported_label_types:Sentence` and adding annotations.
--  Appending the Sentence labels to the created Data.
--  Appending the Data to the created Segment.
+.. code:: python
 
-See :ref:`this page <supported_label_types:Sentence>` for more details for about Sentence annoation
-details.
+    data = dev_segment[0]
 
 .. note::
-   The :ref:`THCHS-30 dataloader <THCHS30-dataloader>` above uses relative import(L13-15).
-   However, when you write your own dataloader you should use regular import as shown below.
-   And when you want to contribute your own dataloader, remember to use relative import.
-
-.. code:: python
-
-   >>> from graviti.dataset import Data, Dataset
-   >>> from graviti.label import LabeledSentence, SentenceSubcatalog, Word
 
-***********************************
- Upload Local Dataset to TensorBay
-***********************************
+    If the :ref:`reference/dataset_structure:Segment` or
+    :ref:`advanced_features/fusion_dataset/fusion_dataset_structure:fusion segment`
+    is created without given name, then its name will be "".
 
-Once you write your own :ref:`contribution:Dataloader` and read the local dataset into a
-:ref:`basic_concepts:Dataset` object, you can upload it to TensorBay and share with the community.
+In each :ref:`reference/dataset_structure:Data`,
+there is a sequence of :ref:`reference/label_format:Sentence` annotations.
+You can get one by index.
 
 .. code:: python
 
-   >>> dataset = gas.create("THCHS-30")
-   >>> dataset_client = gas.upload_dataset(dataset, jobs=8, skip_uploaded_files=False)
-   >>> dataset_client.commit("THCHS-30")
+    labeled_sentence = data.label.sentence[0]
+    sentence = labeled_sentence.sentence
+    spell = labeled_sentence.spell
+    phone = labeled_sentence.phone
 
-Remember to execute the :ref:`features/version_control:Commit` step after uploading.
-If needed, you can re-upload and commit again.
-Please see :ref:`features/version_control:Version Control` for more details.
+There is only one label type in "THCHS-30" dataset, which is ``Sentence``.
+The information stored in  ``sentence``, ``spell`` and ``phone`` of :ref:`reference/label_format:Sentence`
```

Comment:
It contains ``sentence``, ``spell`` and ``phone`` information.
See ... for more details about ....